paddle/scripts/docker/README.md

```diff
@@ -58,7 +58,7 @@ Users can specify the following Docker build arguments with either "ON" or "OFF"
 | `WITH_AVX` | OFF | Set to "ON" to enable AVX support. |
 | `WITH_TESTING` | OFF | Build unit tests binaries. |
 | `WITH_MKL` | ON | Build with [Intel® MKL](https://software.intel.com/en-us/mkl) and [Intel® MKL-DNN](https://github.com/01org/mkl-dnn) support. |
-| `WITH_GOLANG` | ON | Build fault-tolerant parameter server written in go. |
+| `WITH_GOLANG` | OFF | Build fault-tolerant parameter server written in go. |
```

Comment:
May also change `paddle/scripts/docker/build.sh` to make this default OFF

Reply:
https://github.com/PaddlePaddle/Paddle/blob/a67cebaf5f0c7657fc2f773360aeedb396a87d5f/paddle/scripts/docker/build.sh#L43

If changing the above value to `OFF`, then CI cannot test the go codes? Maybe I should change the value in `README.md` back to `ON`?

Reply:
We can configure CI to enable golang explicitly to run the tests, the defaults can be still OFF.

Reply:
Done.